pkg/kubelet/nodestatus/setters.go

```diff
@@ -191,11 +191,11 @@ func NodeAddress(nodeIPs []net.IP, // typically Kubelet.nodeIPs
 
 				if existingHostnameAddress == nil {
 					// no existing Hostname address found, add it
-					klog.Warningf("adding overridden hostname of %v to cloudprovider-reported addresses", hostname)
+					klog.InfoS("adding overridden hostname to cloudprovider-reported addresses", "hostname", hostname)
 					nodeAddresses = append(nodeAddresses, v1.NodeAddress{Type: v1.NodeHostName, Address: hostname})
 				} else if existingHostnameAddress.Address != hostname {
 					// override the Hostname address reported by the cloud provider
-					klog.Warningf("replacing cloudprovider-reported hostname of %v with overridden hostname of %v", existingHostnameAddress.Address, hostname)
+					klog.InfoS("replacing cloudprovider-reported hostname with overridden hostname", "cloudprovider-reportedHostname", existingHostnameAddress.Address, "overriddenHostname", hostname)
```

Comment:
I wonder if we should pick different labels for these... @serathius for replacements in general do we have any guidance on old/new? Might make this a little easier to grok.

cc @dims do you have an opinion?

Reply:
+1 to make it simpler ... say `cloudprovider-hostname` and `override-hostname`

Reply:
There is no strong guidance on prefixes based on context. Some cases like this is are unique enought that we just want to make sure that they make sense in context of this specific log. Of cause if it make sense in context, simpler prefix is better like `old`/`new` > `override`/`cloudprovider`. Don't know this code enough if `old`/`new` makes sense here.

+1 for dims suggestion with simplified prefix, but let's use lower camel case as in migration guide. So
`cloudproviderHostname` and `overrideHostname`

Reply:
+1 from me!